src/sources/opensubtitles.js

```diff
@@ -5,7 +5,7 @@ const request = require("request-promise-native");
 const fs = require("fs");
 const iconv = require("iconv-lite");
 
-const OpenSubtitles = new OS("caption");
```

Comment:
Is "caption" a placeholder?

Reply:
Nope, `caption` is the name of the user agent. We need to keep this since it has been added as an user agent by @opensubtitles.
More information: https://www.npmjs.com/package/opensubtitles-api#quick-start

```js
const OpenSubtitles = new OS({
    useragent: 'caption',
    ssl: true
});
```